README.md

```diff
@@ -6,3 +6,7 @@
 [![Dependency Status](https://gemnasium.com/mmozuras/pronto-reek.png)](https://gemnasium.com/mmozuras/pronto-reek)
 
 Pronto runner for [Reek](https://github.com/troessner/reek), code smell detector for Ruby. [What is Pronto?](https://github.com/mmozuras/pronto)
+
+## Configuration
+
+Configuring Reek via config.reek, or any file ending with .reek, will work just fine with pronto-reek.
```

Comment:
@zewelor could you also link `config.reek` to https://github.com/troessner/reek#configuration-file? 😄